client/packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -238,8 +232,8 @@ export function Autocomplete<T>({
         shouldUseCustomRenderOption ? customRenderOption : undefined
       }
       onChange={(_event, option, reason, details) => {
-        if (isCreateOption(option) && onCreateOptionClick) {
-          onCreateOptionClick(option as AddOption);
+        if (isClickableOption(option) && clickableOption?.onClick) {
+          clickableOption?.onClick();
```

Comment:
silly nit but typescript should be happy to drop the optional chain cause the if condition guarantees it is defined hehe

```suggestion
          clickableOption.onClick();
```

But like... same same lol